frontend/src/pages/Confirmation/validation.js

```diff
@@ -0,0 +1,100 @@
+import Joi from "@hapi/joi";
+
+const validationForIdNamePermissionneeded = Joi.object({
+  id: Joi.string().required(),
+  displayName: Joi.string().required().min(3),
+  listPermissionsNeeded: Joi.boolean().valid(true),
+});
+
+const ValiadtorForIdName = Joi.object({
```

Comment:
Ususally just Class has Capital first letter. Also there is a typo in name.

Reply:
I didn't see it's a typo, I have updated.